docs/README.md

```diff
@@ -7,6 +7,7 @@
 - [Running on azure container services](kubeless-on-azure-container-services.md)
 - [Monitoring](monitoring.md)
 - [Autoscaling](autoscaling.md)
+- [Use existing Kafka cluster](use-existing-kafka.md)
```

Comment:
I think we should also link to this on the main README.md, when talking about pubsub (in your other PR you mention how to install kafka, we should add it there)

Reply:
ok I will update it shortly